Fixes error:
output/host/arm-buildroot-linux-gnueabi/include/c++/5.4.0/type_traits:1459:12: error: invalid use of incomplete type ‘class WebCore::Document’
     struct is_base_of
            ^
In file included from bindings/js/JSDOMWrapper.h:24:0,
                 from ../../DerivedSources/WebCore/JSTreeWalker.h:23,
                 from ./bindings/js/JSTreeWalkerCustom.cpp:21,
                 from ../../DerivedSources/WebCore/unified-sources/UnifiedSource169.cpp:2:
bindings/js/JSDOMGlobalObject.h:39:7: note: forward declaration of ‘class WebCore::Document’
 class Document;
       ^
In file included from ../../DerivedSources/ForwardingHeaders/wtf/Ref.h:34:0,
--
                 from ../../DerivedSources/ForwardingHeaders/JavaScriptCore/Identifier.h:23,
                 from ../../DerivedSources/WebCore/JSDOMBindingInternalsBuiltins.h:34,
                 from ../../DerivedSources/WebCore/WebCoreJSBuiltinInternals.h:39,
                 from bindings/js/JSDOMGlobalObject.h:29,
                 from bindings/js/JSDOMWrapper.h:24,
                 from ../../DerivedSources/WebCore/JSTreeWalker.h:23,
                 from ./bindings/js/JSTreeWalkerCustom.cpp:21,
                 from ../../DerivedSources/WebCore/unified-sources/UnifiedSource169.cpp:2:
../../DerivedSources/ForwardingHeaders/wtf/TypeCasts.h: In instantiation of ‘typename WTF::match_constness<Source, Target>::type& WTF::downcast(Source&) [with Target = WebCore::Document; Source = WebCore::ScriptExecutionContext; typename WTF::match_constness<Source, Target>::type = WebCore::Document]’:
./bindings/js/JSWebAnimationCustom.cpp:47:49:   required from here
../../DerivedSources/ForwardingHeaders/wtf/TypeCasts.h:82:80: error: ‘value’ is not a member of ‘std::is_base_of<WebCore::ScriptExecutionContext, WebCore::Document>’
     return static_cast<typename match_constness<Source, Target>::type&>(source);
                                                                                ^
../../DerivedSources/ForwardingHeaders/wtf/TypeCasts.h:82:79: error: invalid static_cast from type ‘WebCore::ScriptExecutionContext’ to type ‘WTF::match_constness<WebCore::ScriptExecutionContext, WebCore::Document>::type& {aka WebCore::Document&}’
     return static_cast<typename match_constness<Source, Target>::type&>(source);
                                                                               ^
Source/WebCore/CMakeFiles/WebCore.dir/build.make:7759: recipe for target 'Source/WebCore/CMakeFiles/WebCore.dir/__/__/DerivedSources/WebCore/unified-sources/UnifiedSource169.cpp.o' failed
make[5]: *** [Source/WebCore/CMakeFiles/WebCore.dir/__/__/DerivedSources/WebCore/unified-sources/UnifiedSource169.cpp.o] Error 1

diff --git a/Source/WebCore/bindings/js/JSWebAnimationCustom.cpp b/Source/WebCore/bindings/js/JSWebAnimationCustom.cpp
index 8b56f86c151..b8716be0f1c 100644
--- a/Source/WebCore/bindings/js/JSWebAnimationCustom.cpp
+++ b/Source/WebCore/bindings/js/JSWebAnimationCustom.cpp
@@ -26,6 +26,7 @@
 #include "config.h"
 #include "JSWebAnimation.h"
 
+#include "Document.h"
 #include "JSAnimationEffect.h"
 #include "JSAnimationTimeline.h"
 #include "JSDOMConstructor.h"
